2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,11 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let allCurrency = {};
+  for (let [country, currencyCode] of countryCurrencyCodes) {
```

Comment:
I like the way you refer to the properties (country, currencyCode) here rather than the whole object (countryCurrencyCode), this allows you to only need 1 line of code inside the for loop.